src/CADL.Extension/Emitter.Csharp/src/lib/cadlServer.ts

```diff
@@ -71,7 +71,9 @@ export function resolveServers(
                     IsNullable: false
                 } as InputPrimitiveType,
                 Location: RequestLocation.Uri,
-                IsApiVersion: false,
+                IsApiVersion:
+                    name.toLowerCase() === "apiversion" ||
```

Comment:
can we use the similar API here:   isApiVersion(program, parameter);

Reply:
No. current isApiVerison only work on HttpOperationParamter. it is not work on ModelType/ModelProperty. I will ask azure-dpg to support this. But now, we only can calculate ourself. When the api is ready, we will update.